web_app/db/README.md

```diff
@@ -0,0 +1,19 @@
+## Populating the Database with Fake Data
+
+To populate the database with fake data for testing purposes, follow these steps:
+
+1. **Install Dependencies**: Ensure you have the required packages installed. You can do this by running:
+
+   ```bash
+   pip install sqlalchemy psycopg2 faker
```

Comment:
we don't use on the project `pip`, please add these dependencies add to pyproject.toml 
most of them already there

Reply:
okay sir